code/modules/lighting/lighting_profiler.dm

```diff
@@ -0,0 +1,41 @@
+// Writes lighting updates to the database.
+// FOR DEBUGGING ONLY!
+
+var/DBQuery/lprof_q
```

Comment:
This doesn't need to exist as a global variable. Write it as a static one inside lprof_write().